docs/README.md

```diff
@@ -1,6 +1,9 @@
 # Build instructions
 
-To build the docs you must use Python 3.11.
+To build the docs you must install the following requirements:
+
+- GNU Make
```

Comment:
Not necessary on Windows

Reply:
@JeanChristopheMorinPerso Ah, I see you have a `make.bat` that I didn't see the first time in the `docs` directory. I've been just using the `Makefile` on Windows. I'll update the documentation here to mention that you only need GNU Make on Linux/Mac, and that if you're on Windows, to just use the `make.bat` file.